rnote-compose/src/style/textured/mod.rs

```diff
@@ -95,6 +95,16 @@ impl Composer<TexturedOptions> for Line {
     }
 }
 
+impl Composer<TexturedOptions> for Arrow {
```

Comment:
I'd remove this implementation entirely (because there is no actual implementation) and write `unimplemented!()` in `style/mod.rs` similar to how it is done for other shapes that cannot be drawn with the textured style

Reply:
aye aye!